project/firstproject/src/main/java/com/example/firstproject/controller/HelloController.java

```diff
@@ -0,0 +1,15 @@
+package com.example.firstproject.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController
+{
+    @GetMapping("/helloworld")
+    public String helloworld()
```

Comment:
By convention, we should use **camelCase** for java method naming , instead of `helloworld` change it to `helloWorld`